src/components/Button/index.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import BsButton from 'react-bootstrap/Button';
+
+import styles from './styles.module.scss';
+
+import Button from '.';
+
+describe(__filename, () => {
+  const render = (overrides = {}) => {
+    const props = {
+      onClick: jest.fn(),
+      ...overrides,
+    };
+    return shallow(<Button {...props}>example-children</Button>);
+  };
+
+  it('renders a button and children', () => {
+    const root = render({
+      className: 'example-className',
```

Comment:
It would involve less duplication of hardcoded values if you put `className`, `title` and `aria-controls` into constants and then used them in both the `render` function and the assertions.